### ISSUE TITLE
High cpu usage MacOS

### ISSUE THREAD

**Issue by MichelDiz** [Microsoft/vscode#45871]
- VSCode Version:
1.21.1
- OS Version:
10.13.2 H.Sierra

Steps to Reproduce:

1. Open 4 Projects in VS.
2. Let it Open without use.

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
@MichelDiz Can you follow this wiki and post the results back here to understand more about why performance is so bad for you: https://github.com/Microsoft/vscode/wiki/Performance-Issues

Thanks!

**Comment by MichelDiz** [comment]
Many thanks for the reply.

I wonder, would not it be nice to have in VScode a visualization of the internal processes of the program itself? like the task view you have in Chrome. Since it is an application made in electron. You're running over a Chromium, right? There is some support for that.

A "htop" task viewer. Would be great. For us users to watch on the day. No need to take that much time and already point the bug to the correct developer. As an example of an extension.

**Comment by bpasero** [comment]
@MichelDiz we have that, it is all documented in that wiki

**Comment by MichelDiz** [comment]
I got it. But it's not a friendly process like Chrome's. You have an option inside UI to see NPM Tasks and others. Via command there is this possibility, but I tried to do it now. But with code --status "zsh: command not found: code" is not recognized in my terminal. I'll have to see my Bash profile to sort it out.

My issue has not been resolved yet. I'll close this issue anyway. I just downloaded the Code Insiders, I'll test with it.

Thanks again.
